package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "constructor-hangman",
+  "version": "1.0.0",
+  "description": "",
+  "main": "cli.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
I would also start making scripts so that you can just enter commands and have your app run instead of having to remember whatever flags and environment variables you might need to perform certain actions.